### PR TITLE
Be more explicit about DCO

### PR DESCRIPTION
**Pull request by hdost**
* Add link about the browser extension.

Relates #2118

Signed-off-by: Harold Dost <github@hdost.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Prometheus uses GitHub to manage reviews of pull requests.
 
+* Every commit *MUST* signed off on the [DCO](https://github.com/dcoapp/app#how-it-works)
```
Comment (SuperQ) [inline CONTRIBUTING.md:5]: Expand the acronym to make it a little more SEO, screen reader, and mobile screen friendly.

```suggestion
* Every commit *MUST* signed off on the [Developer Certificate of Origin](https://github.com/dcoapp/app#how-it-works) (DCO).
```

**Review by SuperQ — COMMENTED** [review]
Thanks, this is a good improvement.

**Comment by hdost** [comment]
@juliusv  Something appears to be going wrong with Netlify testing.

File: CONTRIBUTING.md (hunk 1 above)
Comment (juliusv) [reply CONTRIBUTING.md:5]: Should it really be "on the" not "with the"?

**Comment by juliusv** [comment]
@hdost Ah drat, I believe that's due to the Netlify setting to omit sensitive environment variables for forks:

variables

Not sure what to do there as a quick fix, because we can't just expose sensitive vars to the world, but the GitHub token is required for assembling the docs from the other repos.

**Comment by juliusv** [comment]
Btw., I think this is the same CONTRIBUTING.md as in the other repos, so we probably want this change everywhere.

I thought it was part of the files that we automatically sync across repos, but it seems CONTRIBUTING.md is not included in that: https://github.com/prometheus/prometheus/blob/707600d84f5567354ef48d4593ddab84c4837811/scripts/sync_repo_files.sh#L40 - @SuperQ @roidelapluie should it be?

**Comment by SuperQ** [comment]
@juliusv Yes, I think we should sync this file. But some repos have repo-specific instructions. We'll have to come up with pattern to handle this. Either move the repo-specific stuff to a separate file, or have some kind of "# Additional considerations` section that we can patch around.

File: CONTRIBUTING.md (hunk 1 above)
Comment (hdost) [reply CONTRIBUTING.md:5]: Agreed

Comment (hdost) [reply CONTRIBUTING.md:5]: > Should it really be "on the" not "with the"?

So I believe "on the" does actually could make sense

But maybe
"Every commit *MUST* be signed off to agree to the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). [How does it work?](https://github.com/dcoapp/app#how-it-works)"

**Comment by hdost** [comment]
> @hdost Ah drat, I believe that's due to the Netlify setting to omit sensitive environment variables for forks:
> 
> 
> 
> variables
> 
> 
> 
> Not sure what to do there as a quick fix, because we can't just expose sensitive vars to the world, but the GitHub token is required for assembling the docs from the other repos.

I'm not so sure because I just had this PR merged yesterday https://github.com/prometheus/docs/pull/2138

**Comment by juliusv** [comment]
> I'm not so sure because I just had this PR merged yesterday https://github.com/prometheus/docs/pull/2138

Ah indeed, I retried the build and now it succeeded.

> @juliusv Yes, I think we should sync this file. But some repos have repo-specific instructions. We'll have to come up with pattern to handle this. Either move the repo-specific stuff to a separate file, or have some kind of "# Additional considerations` section that we can patch around.

Oh, I wasn't aware of some repos having more specific instructions. That complicates things... I mean for now I'd be ok just merging this one here then, if other files are diverging already as well.

File: CONTRIBUTING.md (hunk 1 above)
Comment (juliusv) [reply CONTRIBUTING.md:5]: Yeah, that sounds even better!

**Comment by hdost** [comment]
@juliusv Looks good to go, and I could always push PRs to some of the others with this same language. Just let me know 👍🏼

**Comment by juliusv** [comment]
@hdost Definitely not gonna say no to that, but we do have a lot of repos and that would be creating a lot of work for you that seems like it should be automated in *some* way :)

**Comment by SuperQ** [comment]
We need to resolve at least some of the wording diff with `prometheus/prometheus`. There appear to have been a number of small changes.

**Comment by hdost** [comment]
> We need to resolve at least some of the wording diff with `prometheus/prometheus`. There appear to have been a number of small changes.

Yea `prometheus/prometheus` is significantly different. I'm not sure the others (more code based) would suffer as much from the same issues as docs are something I think more people are likely to not feel they need to try changes locally first.

**Comment by RichiH** [comment]
On mobile.

There's also a "be" missing around the DCO link.


Sent by mobile; please excuse my brevity.

**Pull request merged**